sklearn/grid_search.py

```diff
@@ -670,9 +670,20 @@ class GridSearchCV(BaseSearchCV):
     fit_params : dict, optional
         Parameters to pass to the fit method.
 
-    n_jobs : int, default=1
-        Number of jobs to run in parallel.
-
+    n_jobs: int, default: 1 :
+        The maximum number of concurrently running jobs, such as the number of 
+        Python worker processes when backend=”multiprocessing” or the size of 
```

Comment:
The user cannot really choose the backend here so this is too much information, I think.

Perhaps copy an n_jobs description from elsewhere in scikit-learn, rather than from joblib.parallel.

Also, please replicate your changes to `RandomizedSearchCV`.